scenarios/Assistants/multi-agent_example/README.md

```diff
@@ -0,0 +1,139 @@
+# Azure OpenAI Multi-Agent Assistant API
```

Comment:
Missing YAML frontmatter for discovery on https://learn.microsoft.com

https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md#discoverability